packages/reports/src/Components/Form/CompositeCalendar/index.js

```diff
@@ -1,3 +1 @@
-import CalendarIcon from './calendar-icon';
-
-export default CalendarIcon;
+export default from './composite-calendar.jsx';
```

Comment:
We had eslint errors for this pattern so you should not use it
```suggestion
import CompositeCalendar from  './composite-calendar.jsx';

export default CompositeCalendar
```